Mathlib/RingTheory/Ideal/Quotient.lean

```diff
@@ -104,6 +104,9 @@ def mk (I : Ideal R) : R →+* R ⧸ I where
   map_add' _ _ := rfl
 #align ideal.quotient.mk Ideal.Quotient.mk
 
+instance {I : Ideal R} : CoeTC R (R ⧸ I) :=
```

Comment:
Could you add a justification about why "Users should generally not implement this directly." does not apply here?

Reply:
TBH looking at https://github.com/leanprover/lean4/blob/master/src/Init/Coe.lean#L61 it looks to me that just a simple `Coe` is ok here and that `CoeTC` is actually more conservative in that it probably wont allow chaining if implemented directly

Reply:
So the following doesn't  work with `CoeTC` but does with `Coe`
```
variable (x : R) (I : Ideal R) (J : Ideal (R ⧸ I))
#check (x : (R⧸ I) ⧸J) 
```